Why: same as patches/mono-1.2.6/0006 (still needed in 2.6.4): 
mono-profiler-aot.c tests PLATFORM_WIN32 to pick the one-argument Windows 
mkdir() but does not include config.h.

diff --git a/mono/profiler/mono-profiler-aot.c b/mono/profiler/mono-profiler-aot.c
index 92285bbc..c54c6a91 100644
--- a/mono/profiler/mono-profiler-aot.c
+++ b/mono/profiler/mono-profiler-aot.c
@@ -1,3 +1,4 @@
+#include <config.h>
 /*
  * mono-profiler-aot.c: Ahead of Time Compiler Profiler for Mono.
  *
